src/extractr.js

```diff
@@ -40,11 +46,13 @@ export default class Extractr {
     const prev = this.avgs[feature]
 
     this.avgs[feature] = prev * a + (1 - a) * curr
-
+    
```

Comment:
we need to set up a `npm run prettier` script, though I think you can just run `prettier`.